SU-HSDO/suhumsci
# Fix: Revert Spotlight

# READY FOR REVIEW

## Summary
Since Spotlight is not quite in a place that H&S is comfortable with for old themes, such as su_humsci theme, we are reverting all changes so that the Spotlight does not go into Prod this week.

I also recommited the compiled CSS as it will have changed.

✅ All codeclimate tests pass: https://codeclimate.com/github/SU-HSDO/suhumsci/pull/647 against develop.

Because I used `git revert` we will be able to keep the commits with the history of the Spotlight earlier in the branch. In the future when we are ready to deploy the Spotlight, we can make a new PR that reverts just this commit!

## Need Review By (Date)
7/8

## Urgency
high

## Steps to Test
1. `npm run test` in humsci_basic passes
2. Ensure that all Spotlight features and config are removed from 8.2.13-release.
3. Run `lando drush @economics.local config-import` or do a sync (I had to) and then run `lando drush @economics.local cr` to make sure that Spotlight is not an option available on the basic page in the hero or components region anymore.

<img width="681" alt="Screen Shot 2020-07-07 at 3 54 36 PM" src="https://user-images.githubusercontent.com/12848168/86835625-631af980-c06a-11ea-99cf-e9d4a9e0ea67.png">


## PR Checklist
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
- [Sparkbox PR Checklist](../docs/SparkboxPRChecklist.md)
